classes_ja.js

```diff
@@ -0,0 +1,50 @@
+//////  1  /////////////
+
+class Human {
+  constructor(name, age) {
+    this.name = name;
+    this.age = age;
+  }
+  ageOneYear() {
+    return this.age++;
+  }
+  eatin() {
+    return `I ${this.name} Pahh, dah, paah pah pahhh, I'm lovin' it....`;
+  }
+}
+
+let pancho = new Human("Panchito", 35);
+
+console.log(pancho.age);
+pancho.ageOneYear();
+console.log(pancho.age);
+console.log(pancho.eatin());
+///////  2   ////////////////
+class Vector {
+  constructor(x, y) {
+    this.x = x;
+    this.y = y;
+  }
+  add(vec) {
+    let newx = this.x + vec.x,
+      newy = this.y + vec.y;
+    return [newx, newy];
+  }
+  minus(vec) {
+    let newx = this.x - vec.x,
+      newy = this.y - vec.y;
+    return [newx, newy];
+  }
```

Comment:
The plus and minus functions should return a new Vector object, not an array with the results.